easybuild/easyconfigs/x/x265/x265-1.9-goolf-1.7.20.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'x265'
+version = '1.9'
+
+homepage = 'http://www.videolan.org/developers/x265.html'
+description = """x265 is a free software library and application for encoding video streams into the H.265
+ AVC compression format, and is released under the terms of the GNU GPL."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['http://ftp.videolan.org/pub/videolan/x265/']
+sources = ['x265_%(version)s.tar.gz']
+
+builddependencies = [('CMake', '2.8.11')]
```

Comment:
@eediaz1987 you will need to bump this version to `3.5.2`, since we only have an easyconfig available for that version of CMake with `goolf.1.7.20`